lib/exprotobuf.ex

```diff
@@ -4,6 +4,7 @@ defmodule Protobuf do
   alias Protobuf.Config
   alias Protobuf.ConfigError
   alias Protobuf.Field
+  alias Protobuf.OneofField
```

Comment:
I would rather name this `OneOfField` instead of `OneofField` for readability sake.